content/docs/self-hosted-runners.md

```diff
@@ -73,12 +73,27 @@ CML and its dependencies setup to use CML functions like `cml-send-comment` from
 your instance, you can create your favorite training environment in the cloud by
 pulling the Docker container of your choice.
 
-We like the
-[CML container](https://github.com/iterative/cml/blob/master/Dockerfile)
+We like the [CML Docker images](#docker-images)
 (`docker://iterativeai/cml:0-dvc2-base1`) because it comes loaded with Python,
 Git, Node JS and other essentials for full-stack data science. But we don't mind
 if you do it your way :)
 
+## Docker Images
+
+The CML Docker images (`docker://iterativeai/cml` or
+`docker://ghcr.io/iterative/cml`) come loaded with Python, CUDA, `git`, `node`
+and other essentials for full-stack data science. Different versions of these
+essentials are available from different `iterativeai/cml` image tags. The tag
+convention is `{CML_VER}-dvc{DVC_VER}-base{BASE_VER}{-gpu}`:
```

Comment:
When I read this automatically I can think of ```0.6.2-dvc2.1.0-base1-gpu```
Maybe this can be expressed in another way?

Reply:
Perhaps `{CML_MAJOR}` and so on?